### PR TITLE
Design overhauled (v1)

### PR DESCRIPTION
**Pull request by MartynasAnn**
Changed the following:

-New flyout and login styles implementing app's logo
-Changed menu icon to contain the app's logo
-Changed flyout icons to non-generic ones fitting the theme
-Changed placement and dimensions of various buttons
-Rounded the buttons. All of them.
-Created custom resources with logo and app name to use as splash screen (upcoming in v2)
-Fixed fonts to better match the new design
-Primary and secondary colors now defined
-Created color scheme of the app to be used in further design improvements
-Improved overall feel of the app (I hope)

To be added in v2:

-Splash screen
-Animations
-Profile page overhaul (prettify stats to fit the rounded theme, fix background)
-Shadows and more
-Remove city label, since it's pretty much redundant at given time

### CODE REVIEW

File: EncounterMeApp/EncounterMeApp/Views/ProfilePage.xaml.cs [hunk 1]
```diff
@@ -45,7 +47,15 @@ private async void btnProfilePicture_Clicked(object sender, EventArgs e)
         {
             string profilePicPath = await DisplayPromptAsync("What is the file name?", "(The file must exist in Resources/drawable folder of the app", keyboard: Keyboard.Default);
             ProfileImage.Source = profilePicPath;
+            //var result = MediaPicker.PickPhotoAsync(new MediaPickerOptions
```
Comment (Dominic-dul) [inline EncounterMeApp/EncounterMeApp/Views/ProfilePage.xaml.cs:50]: Is this comment really necessary?

**Review by Dominic-dul — APPROVED** [review]
Wow. Seems like a huge job to do overnight. Can't spot any major errors or drawbacks. Maybe comments of previous code versions, where you use color schemas, could be deleted because they may be misleading in the future. Overall great job!

File: EncounterMeApp/EncounterMeApp/Views/ProfilePage.xaml.cs (hunk 1 above)
Comment (MartynasAnn) [reply EncounterMeApp/EncounterMeApp/Views/ProfilePage.xaml.cs:50]: Whoops, seems like a remnant from profile photo picker, will be deleted :)

**Review by PijusPale — APPROVED** [review]
What a clutch! Looking to see the overall view of the app :)

**Pull request merged**